### PR TITLE
added error404 page on incorrect routes

### LINKED ISSUE #64: No Error404 Page on Incorrect Routes
While exploring the codebase, I found when someone is going to an incorrect routes, they are getting an empty page! Hence, there must be a UI to show Error 404 and should give a button to go back to the Home Page for the easy of university students. 

<br>

image

<br> 

I will look and work on this issue on my own. Thanks!

### PR DESCRIPTION
**Pull request by p1yu5h0**
Fixes #64 

As an errorPage404 is important element for a website (discussed in #64)

Changes Done:-
1) Added an extra Route path `/*` and displaying <Error404 /> page.
2) Developed and designed <Error404 /> so that the user don't get an empty page on an incorrect route.

image

### CODE REVIEW

**Comment by NikolasPpd** [comment]
We should have support for internationalization (using i18n) on this page like we do on the rest of the app.

**Comment by p1yu5h0** [comment]
Sure @NikolasPpd I will do this and update the PR. Thanks!

**Comment by p1yu5h0** [comment]
Hi @NikolasPpd I have made the changes suggested by you, added support for internationalisation (using i18n). Please review! 
Thanks, Piyush.
image
image

File: src/assets/locales/el/error404.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line
+export default {
+    error_404: "Σφάλμα 404",
+    page_not_found: "Η σελίδα δεν βρέθηκε",
+    go_to_homepage: "ΠΗΓΑΙΝΕ ΣΤΗΝ ΑΡΧΙΚΗ ΣΕΛΙΔΑ",
```
Comment (NikolasPpd) [inline src/assets/locales/el/error404.js:5]:
```suggestion
    go_to_homepage: "Πήγαινε στην αρχική σελίδα",
```

**Review by NikolasPpd — CHANGES_REQUESTED** [review]

**Review by NikolasPpd — APPROVED** [review]

File: src/assets/locales/el/error404.js (hunk 1 above)
Comment (p1yu5h0) [reply src/assets/locales/el/error404.js]: Thanks for this suggestion Nikolas, I have committed the changes.

**Comment by p1yu5h0** [comment]
@GeorgeApos @GeorgeFkd Can you please approve and merge these changes? 
Thanks, Piyush

**Pull request merged**